packages/mui-material/src/Slider/Slider.js

```diff
@@ -470,7 +470,7 @@ const useUtilityClasses = (ownerState) => {
       track === 'inverted' && 'trackInverted',
       track === false && 'trackFalse',
       color && `color${capitalize(color)}`,
-      size && `size${capitalize(size)}`,
+      size !== 'medium' && `size${capitalize(size)}`,
```

Comment:
I'll revert this change to be extra careful if anyone uses this class currently. Enforcing the standard will come in v6: https://github.com/mui/material-ui/issues/38598

```suggestion
      size && `size${capitalize(size)}`,
```